app/models/automated_checks/claim_verifiers/identity.rb

```diff
@@ -42,8 +42,18 @@ def complete_match
         create_task(match: :all, passed: true)
       end
 
-      def create_field_note(name:)
-        body = "#{name} not matched"
+      def create_field_note(
+        name:,
+        claimant:,
+        dqt:
+      )
+        body = <<~HTML
+          #{name} not matched:
+          <pre>
+            Claimant: <span class="dark-grey">"</span><span class="red">#{claimant}</span><span class="dark-grey">"</span>
+            DQT:      <span class="dark-grey">"</span><span class="green">#{dqt}</span><span class="dark-grey">"</span>
```

Comment:
This feels presentational, not sure why in model.

Reply:
Speed. Refactor later 👍